chia/pools/pool_wallet.py

```diff
@@ -313,11 +309,6 @@ async def rewind(self, block_height: int) -> bool:
             await self.wallet_state_manager.pool_store.rollback(block_height, self.wallet_id)
 
             if len(history) > 0 and history[0][0] > block_height:
-                # If we have no entries in the DB, we have no singleton, so we should not have a wallet either
-                # The PoolWallet object becomes invalid after this.
-                await self.wallet_state_manager.interested_store.remove_interested_puzzle_hash(
```

Comment:
Why not remove the puzzle hash? The pool wallet dissapears here if the creation TX is reorged

Reply:
Not using puzzle hash here at all anymore. (it's not added)